software/react-native/operations.md

```diff
@@ -2,8 +2,39 @@
 
 #### Deployment Strategy - Paid (Preferred)
 
-Buddybuild[https://www.buddybuild.com/] is the recommended deployment toolchain.
+[Buddybuild](https://www.buddybuild.com/) is the recommended deployment toolchain.
 
 #### Deployment Strategy - Open Source
 
-Fastlane[https://fastlane.tools/] is the open source alternative to deployment.
+[Fastlane](https://fastlane.tools/) is the open source alternative to deployment.
+
+
+#### Testing - Jest
```

Comment:
would it be worth talking about Ava at all anymore? Not sure if thats still something to discuss